lib/filewatch/tail.rb

```diff
@@ -226,23 +227,11 @@ def _sincedb_open
     private
     def _sincedb_write
       path = @opts[:sincedb_path]
-      tmp = "#{path}.new"
-      begin
-        db = File.open(tmp, "w")
-      rescue => e
-        @logger.warn("_sincedb_write failed: #{tmp}: #{e}")
-        return
-      end
-
-      @sincedb.each do |sincedb_record_uid, pos|
-        db.puts([sincedb_record_uid, pos].flatten.join(" "))
-      end
-      db.close
-
-      begin
-        File.rename(tmp, path)
-      rescue => e
-        @logger.warn("_sincedb_write rename/sync failed: #{tmp} -> #{path}: #{e}")
+      return if path == "/dev/null"
```

Comment:
Would be nice, although since that won't work well on windows and /dev/null is such a special case..I'll investigate further